test/Cookbook-test.js

```diff
@@ -1,30 +1,61 @@
 import { expect } from 'chai';
 import Cookbook from '../src/classes/Cookbook';
+import Recipe from '../src/classes/Recipe';
 import ingredientTestData from '../src/data/testing-data/ingredients-test-data'
 import recipeTestData from '../src/data/testing-data/recipe-test-data'
 
 
 describe('Recipe', () => {
-  let cookbook
+  let recipes;
+  let cookbook;
 
   beforeEach(() => {
-
+    let recipeData = recipeTestData.recipeData;
+    recipes = recipeData.reduce((sum, recipe) => {
+      sum.push(new Recipe(recipe));
+      return sum;
+    }, []);
+    cookbook = new Cookbook(recipes)
   })
   
   it('Should be a function', () => {
     expect(Cookbook).to.be.a('function');
   });
-
-  it('should be a function')
   
-  it('should be an instance of RecipeRepository')
+  it('should be an instance of Cookbook', () => {
+    expect(cookbook).to.be.an.instanceOf(Cookbook)
+  })
 
-  it('should be able to filter by 1 tag')
+  it('should store recipes', () => {
+    expect(cookbook.recipes).to.be.an('array');
+    expect(cookbook.recipes[0]).to.be.an.instanceOf(Recipe)
+  })
 
-  it('should be able to filter by 2 or more tags')
+  it('should be able to filter by 1 tag', () => {
+    cookbook.filterByTag('snack')
+    expect(cookbook.currentRecipes.length).to.deep.equal(2)
+  })
 
-  it('should be able to filter by name')
+  it('should be able to filter by 2 or more tags', () => {
+    cookbook.filterByTag('snack','lunch')
+    expect(cookbook.currentRecipes.length).to.deep.equal(1)
+  })
 
-  it('should be able to filter by ingredient')
+  it('should be able to filter by name', () => {
+    cookbook.filterByKeyword('Loaded')
```

Comment:
We will need to remember to convert all incoming data to whatever case sensitivity is needed to compare in the dataset